src/models/Comment.js

```diff
@@ -0,0 +1,38 @@
+import Sequelize from 'sequelize';
+import db from './db'
+
+const Comment = db.define('comment', {
+  qno: {
+    type: Sequelize.INTEGER,
+    allowNull: false,
+    unique: true,
+    validate: {
+      notEmpty: true,
+      isInt: true,
+    }
+  },  
+  comment: {
+    type: Sequelize.STRING,
+    allowNull: true,
+    validate: {
+      notEmpty: false,
+    }
+  },
+  time: {
```

Comment:
Remove this, no need since sequelize stores it by default

Reply:
but the issue says to store the time stamp

Reply:
It is automatically stored by sequelize